### PR TITLE
Fork and Clone a Repository

### PR DESCRIPTION
**Pull request by Remakh**
# Pull Request Template

**script name** - Fork and Clone

**A script that forks and clones a repository onto your device, made using PyGithub.**

### Self Check(Tick After Making pull Request)

- [x] This issue was assigned to me.
- [x] One Change in one Pull Request
- [x] My file is in proper folder (Name of folder should be in lowercase with no space in between)
- [x] I am following clean code and Documentation and my code is well linted with flake8.
- [x] I have added README.md and requirements.txt with my script

If issue was not assigned to you Please don't make a PR. It will marked as invalid.

### CODE REVIEW

**Comment by m4dummies** [comment]
Hi @Remakh  are u having problems with linter?

**Comment by Remakh** [comment]
> Hi @Remakh are u having problems with linter?

I had some issues but I got it in the end.

**Review by Sacrezar — CHANGES_REQUESTED** [review]
Hey, thank you for your contribution. Here is my review:

Forking is working great but you'll have to change your way to authenticate, see: https://developer.github.com/changes/2020-02-14-deprecating-password-auth/

There is some issue as for the cloning, I have the following error:
```ERROR: clone() takes 1 positional argument but 2 were given```

If you can, it would be interesting to allow the user to choose between SSH or http connexion when cloning. 

It is not an obligation but it could be cool to let the user hardcode his token, the default path and the default way to clone (ssh, http or git cli). Either hardcode it or store it in a json file.
That way, it opens the possibility to do something like that:
```
To fork into your account:
py Fnc.py python-geeks/Automation-scripts

To fork into your organization:
py Fnc.py python-geeks/Automation-scripts my_organization
```
See sys.argv [here](https://docs.python.org/3/library/sys.html#sys.argv) and [there](https://www.geeksforgeeks.org/how-to-use-sys-argv-in-python/).

For the requirements.txt, could you just do these little changes please? 
* Remove `git = 2.20.0`
* Change `PyGithub = 1.5.3` for `PyGithub == 1.53` or just `PyGithub`

**Comment by Remakh** [comment]
> Hey, thank you for your contribution. Here is my review:
> 
> Forking is working great but you'll have to change your way to authenticate, see: https://developer.github.com/changes/2020-02-14-deprecating-password-auth/
> 
> There is some issue as for the cloning, I have the following error:
> `ERROR: clone() takes 1 positional argument but 2 were given`
> 
> If you can, it would be interesting to allow the user to choose between SSH or http connexion when cloning.
> 
> It is not an obligation but it could be cool to let the user hardcode his token, the default path and the default way to clone (ssh, http or git cli). Either hardcode it or store it in a json file.
> That way, it opens the possibility to do something like that without any other interaction needed:
> 
> ```
> To fork into your account:
> py Fnc.py python-geeks/Automation-scripts
> 
> To fork into your organization:
> py Fnc.py python-geeks/Automation-scripts my_organization
> ```
> 
> See sys.argv [here](https://docs.python.org/3/library/sys.html#sys.argv) and [there](https://www.geeksforgeeks.org/how-to-use-sys-argv-in-python/).
> 
> For the requirements.txt, could you just do these little changes please?
> 
> * Remove `git = 2.20.0`
> * Change `PyGithub = 1.5.3` for `PyGithub == 1.53` or just `PyGithub`

Thanks for all your suggestions! I'll try to implement them and make the changes

**Comment by Sacrezar** [comment]
No worries.

If you have any problem with flake8 linting, you can install it via 
```
py -m pip install flake8
```
and run it like this:
```
py -m flake8 my_file.py
```

Alternatively, if you're using vscode, you can change your linter from pylint to flake8.
CTRL SHIFT P and write linter. 
image
image

File: fork_and_clone/Fnc.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
+import subprocess
+import sys
+from github import Github
+
+
+def fork():
+
```
Comment (pawangeek) [inline fork_and_clone/Fnc.py:7]: Add docstring about input args

File: fork_and_clone/Fnc.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            org.create_fork(g.get_repo(repo))
+        clone(username, repo)
+    except Exception as e:
+        print("ERROR: " + str(e))
+        print("You can try again or click ctrl-c and exit")
+        fork()
+
+
+def clone(repo):
+
```
Comment (pawangeek) [inline fork_and_clone/Fnc.py:35]: Here too

**Review by pawangeek — CHANGES_REQUESTED** [review]

File: fork_and_clone/Fnc.py (hunk 2 above)
Comment (Remakh) [reply fork_and_clone/Fnc.py:35]: Sure will do

**Review by pawangeek — APPROVED** [review]

**Pull request merged**